tests/test_diff_tables.py

```diff
@@ -151,7 +154,7 @@ def test_diff_small_tables(self):
         )
         self.preql.commit()
         diff = list(self.differ.diff_tables(self.table, self.table2))
-        expected = [("-", (2, datetime.datetime(2022, 1, 1, 0, 0)))]
+        expected = [("-", ("2", time + ".000000"))]
```

Comment:
The ints are strings now because we're converting everything to text for every database, yeah?

Reply:
Yeah.